src/styles/index.scss

```diff
@@ -1,3 +1,253 @@
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: rgba(97, 96, 112, 1);
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 14px;
+  }
+
+  &__price-text {
+    line-height: 18px;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__price-number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: rgba(6, 11, 53, 1);
+  }
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+
+  .logo img {
+    display: block;
+  }
+
+  .nav {
+    &__list {
+      margin-top: 0;
+      margin-bottom: 0;
+      padding-left: 0;
+      display: flex;
+      list-style-type: none;
+      position: relative;
+
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 60px;
+      text-decoration: none;
+
+      .nav__link {
+        color: #000;
+
+        &:hover {
+          color: rgba(0, 172, 220, 1);
+        }
+
+        &-is-active {
+          color: rgba(0, 172, 220, 1);
+        }
+
+        &-is-active::after {
+          content: '';
+          position: absolute;
+          height: 4px;
+          width: 37px;
+          background-color: #00acdc;
+          border-radius: 8px;
+          bottom: 0;
+          left: 0;
+        }
+      }
+
+      &_item:not(:last-child) {
+        margin-right: 20px;
+      }
+    }
+  }
+}
+
+/* .header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo-link {
+    display: block;
+  }
+
+} */
+
+/* .links {
+  text-transform: uppercase;
+  font-weight: 500;
+
+  &-list {
+    display: flex;
+    list-style: none;
+
+    & :not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &-link {
+    position: relative;
+    display: block;
+    text-decoration: none;
+    color: rgba(6, 11, 53, 1);
+    padding-block: 23px;
+    transition: 300ms;
+
+    &:hover {
+      color: rgba(0, 172, 220, 1);
+    }
+
+    &-active {
+      color: rgba(0, 172, 220, 1);
+
+      &::before {
+        content: '';
+        display: block;
+        position: absolute;
+        left: 0;
+        bottom: 0;
+        width: 100%;
+        height: 4px;
+        border-radius: 8px;
+        background-color: rgba(0, 172, 220, 1);
+      }
+    }
+  }
+} */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.body {
+  margin: 0;
+}
```

Comment:
The `.body` class is redundant since the `body` element already has a margin reset. Consider removing this class unless it's needed for specific styling.